bench/libphonenumber/optimized/generate_geocoding_data.cc.ll

```diff
@@ -7238,26 +7238,31 @@ _ZN4absl7debian211string_viewC2ISaIcEEERKNSt7__cxx1112basic_stringIcSt11char_tra
   br i1 %.inv.i.i.i, label %.thread, label %_ZNK4absl7debian218container_internal22StringBtreeDefaultLessclENS0_11string_viewES3_.exit.thread
 
 _ZNK4absl7debian218container_internal22StringBtreeDefaultLessclENS0_11string_viewES3_.exit: ; preds = %_ZN4absl7debian211string_viewC2ISaIcEEERKNSt7__cxx1112basic_stringIcSt11char_traitsIcET_EE.exit17, %22
-  %26 = icmp ult i64 %13, %17
-  br i1 %26, label %_ZNK4absl7debian218container_internal22StringBtreeDefaultLessclENS0_11string_viewES3_.exit.thread, label %28
+  %26 = icmp ugt i64 %13, %17
+  %27 = zext i1 %26 to i32
+  %28 = icmp ult i64 %13, %17
+  %.neg.i.i.i = sext i1 %28 to i32
+  %29 = add nsw i32 %.neg.i.i.i, %27
+  %30 = icmp slt i32 %29, 0
```

Comment:
Not sure why this regresses, but it would get fixed by https://github.com/llvm/llvm-project/pull/103833 anyway.